code/lab1-tdd-triangle/src/test/java/com/lahtachev/triangle/TriangleTests.java

```diff
@@ -0,0 +1,341 @@
+package com.lahtachev.triangle;
```

Comment:
> Внимание: просьба все последующие лабораторные выполнять в той же директории с модулем, развивая и наращивая проект. Отдельный папки для каждой лабораторной (вида kirill-korniakov-lab1, kirill-korniakov-lab2 и т.п.) созданы лишь для того, чтобы вы видели разницу между лабораторными. Вы же должны работать в одной папке, выполняя все задания сквозным образом. **Также, не используйте свое имя в названии проекта -- дайте ему имя, соответствующее теме.**

Reply:
Thanks for reminding me about the tests.